website/docs/docs/bi-directional-contract-testing/consumer.md

```diff
@@ -5,29 +5,29 @@ sidebar_label: Consumer
 
 ## Principles
 
-It is important you use a _mock_ and not a _stub_. The crucial difference is that a mock validates behaviour, a stub does not. We need to ensure that your consumer code actually calls the mock, otherwise we will serialise a contract with invalid expectations.
+It is important for to use a _mock_ and not a _stub_. The crucial difference is that a mock validates behaviour, whereas a stub does not. We need to ensure that your consumer code calls the mock, otherwise we will serialise a contract with invalid expectations.
```

Comment:
I think you meant to delete the word 'for' here?

Reply:
good spot, thanks. I started incorporating the docs team changes here but obviously missed this!